debug.log

```diff
@@ -0,0 +1,4 @@
+[0809/081452.221:ERROR:crash_report_database_win.cc(428)] unexpected header
```

Comment:
You probably don't wanna push your debug logs to Git, since it is peculiar to your system.

next time, remember to exclude them by adding ***.log** (for all log files) or **debug.log** (for specific file) on gitignore